Overview.src.html

```diff
@@ -1131,8 +1131,39 @@ <h4 id=terminology-headers>Headers</h4>
  <li>Return <var title>stream</var>.
 </ol>
 
+<p>To <dfn title=concept-construct-ReadableStream-with-byte-stream>construct</dfn> a
+<code title=concept-ReadableStream>ReadableStream</code> with given a byte stream
+<var>bstream</var>, run these steps:
+
+<ol>
+ <li><p>Let <var>strategy</var> be an object. The user agent may choose any object.
+
+ <li>
+  <p>Let <var>pull</var> be an action that runs these substeps:
+
+  <ol>
+    <li><p>Let <var>bytes</var> be a byte sequence which is 
+    <span data-anolis-spec=encoding title=concept-stream-read>read</span> from <var>bstream</var>.
+
+    <li><p>If <span data-anolis-spec=encoding title=concept-stream-read>read</span> returns
+    <span data-anolis-spec=encoding>end-of-stream</span>,
+    <span title=concept-close-ReadableStream>close</span> <var>stream</var> defined below.
```

Comment:
This reads rather weird. Perhaps introduce a flag before these substeps, set it here, and then after these substeps and after `<var>stream</var>` is allocated, use it to close it if the flag is set?